src/libpmemfile-posix/block_array.c

```diff
@@ -196,6 +318,12 @@ relocate_block(struct pmemfile_block *dst, struct pmemfile_block *src)
 	TX_MEMCPY(dst, src, sizeof(*src));
 }
 
+/*
+ * must_remove_a_block_array
+ * Is there a block_array containing no block metadata?
+ * Is this block_array not the one stored inside the pmemfile_vinode struct?
+ * If it is, it can't be deallocated.
```

Comment:
It's not clear to me how this comment applies to function name. Both could be improved...